### PR TITLE
Production script changes for monorepo

### PR DESCRIPTION
**Pull request by zanderz**
Seems like it even built:
https://s3.amazonaws.com/prerelease.keybase.io/logs/keybase.build.windows.log-7ADVOB6XJHFACOSR3PNLJRMNKSGCIIOHMLN6GLW45FQ3AL5DNIQA.txt

### CODE REVIEW

File: packaging/windows/build_prerelease.cmd [hunk 1]
```diff
@@ -22,22 +22,22 @@ go build -a -tags "prerelease production" -ldflags="-X github.com/keybase/client
...
+pushd %GOPATH%\src\github.com\keybase\client\go\kbfs\kbfsdokan
 :: Make sure the whole build fails if we can't build kbfsdokan
 del kbfsdokan.exe
 :: winresource invokes git to get the current revision
-for /f %%i in ('git -C %GOPATH%\src\github.com\keybase\kbfs rev-parse --short^=8 HEAD') do set KBFS_HASH=%%i
-for /f "tokens=1 delims=+" %%i in ("%KEYBASE_BUILD%") do set KBFS_BUILD=%%i+%KBFS_HASH%
-echo KBFS_BUILD %KBFS_BUILD%
+:: for /f %%i in ('git -C %GOPATH%\src\github.com\keybase\kbfs rev-parse --short^=8 HEAD') do set KBFS_HASH=%%i
+:: for /f "tokens=1 delims=+" %%i in ("%KEYBASE_BUILD%") do set KBFS_BUILD=%%i+%KBFS_HASH%
+:: echo KBFS_BUILD %KBFS_BUILD%
```
Comment (jzila) [inline packaging/windows/build_prerelease.cmd]: :fire:

File: packaging/windows/dorelease.cmd [hunk 2]
```diff
@@ -22,7 +22,7 @@ IF [%UpdateChannel%] == [Smoke2] goto:done_ci
 
 :: NOTE: We depend on the bot or caller to checkout client first
 :: call:checkout_keybase client, %ClientRevision% || goto:build_error || EXIT /B 1
-call:checkout_keybase kbfs, %KBFSRevision% || goto:build_error || EXIT /B 1
+:: call:checkout_keybase kbfs, %KBFSRevision% || goto:build_error || EXIT /B 1
```
Comment (jzila) [inline packaging/windows/dorelease.cmd]: :fire:

File: packaging/windows/dorelease.cmd [hunk 3]
```diff
@@ -170,13 +170,13 @@ EXIT /B 1
 
 :check_ci 
 for /f %%i in ('git -C %GOPATH%\src\github.com\keybase\client rev-parse --short^=8 HEAD') do set clientCommit=%%i
-for /f %%i in ('git -C %GOPATH%\src\github.com\keybase\kbfs rev-parse --short^=8 HEAD') do set kbfsCommit=%%i
-echo [%clientCommit%] [%kbfsCommit%]
+:: for /f %%i in ('git -C %GOPATH%\src\github.com\keybase\kbfs rev-parse --short^=8 HEAD') do set kbfsCommit=%%i
```
Comment (jzila) [inline packaging/windows/dorelease.cmd]: :fire:

File: packaging/windows/dorelease.cmd [hunk 4]
```diff
@@ -170,13 +170,13 @@ EXIT /B 1
...
+echo [%clientCommit%]
 :: need GITHUB_TOKEN
 pushd %GOPATH%\src\github.com\keybase\release
 go build || goto:build_error || EXIT /B 1
 release wait-ci --repo="client" --commit="%clientCommit%" --context="continuous-integration/jenkins/branch" --context="ci/circleci"  || goto:ci_error
-release wait-ci --repo="kbfs" --commit="%kbfsCommit%" --context="continuous-integration/jenkins/branch" --context="ci/circleci"  || goto:ci_error
+:: release wait-ci --repo="kbfs" --commit="%kbfsCommit%" --context="continuous-integration/jenkins/branch" --context="ci/circleci"  || goto:ci_error
```
Comment (jzila) [inline packaging/windows/dorelease.cmd]: :fire:

File: packaging/windows/build_prerelease.cmd (hunk 1 above)
Comment (zanderz) [reply packaging/windows/build_prerelease.cmd]: ok

File: packaging/windows/dorelease.cmd (hunk 4 above)
Comment (zanderz) [reply packaging/windows/dorelease.cmd]: ok

File: packaging/windows/dorelease.cmd (hunk 2 above)
Comment (zanderz) [reply packaging/windows/dorelease.cmd]: ok

**Pull request merged**